### PR TITLE
Initial Request templates implementation

### LINKED ISSUE #22: Implement Request templates

### PR DESCRIPTION
**Pull request by SundeepChand**
Fixes #22 

Here is a demo of the implementation

https://user-images.githubusercontent.com/34120932/123240303-d331c200-d4fd-11eb-9fd8-9a039c6e0361.mp4

Goes with [EOS-uiux-Solutions/strapi/pull/8](https://github.com/EOS-uiux-Solutions/strapi/pull/8)

### CODE REVIEW

**Comment by dvijaywargiya** [comment]
LGTM

File: src/pages/NewStory.js [hunk 1]
```diff
@@ -131,13 +157,18 @@ const NewStory = () => {
...
   */
+  const handleProductSelectChange = (event) => {
+    const id = event.target.value
+    const selectedProduct = products.find((product) => product.id === id)
+    setCurrentProductSelected(selectedProduct?.id ?? 'None')
+  }
 
   const onSubmit = async (data) => {
-    if (data.description === undefined || data.description.length === 0) {
+    if (description === undefined || description.length <= 0) {
```
Comment (kbhutani0001) [inline src/pages/NewStory.js:167]:
```suggestion
    if (!description?.length) {
```

File: src/pages/NewStory.js [hunk 2]
```diff
@@ -148,7 +179,7 @@ const NewStory = () => {
                 Description: "${data.description}"
                 Title: "${data.title}"
                 Category: ${data.category}
-                user_story_status: "5f0f33205f5695666b0d2e7e"
+                user_story_status: "60b5cef600971013c4f269c2"
```
Comment (kbhutani0001) [inline src/pages/NewStory.js:182]: what is this value? can we have a map of status somewhere rather than hardcoding these values please?

**Comment by sharmaaditya570191** [comment]
I think it is a better idea to not send user story status from frontend because this is increasing an api call to fetch the id of the user story status. Another approach can be to make the status an optional field in the backend and assign the required status id in `beforeCreate` lifecycle hook in the backend.

**Review by kbhutani0001 — APPROVED** [review]

**Pull request merged**